style.css

```diff
@@ -0,0 +1,313 @@
+.first {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+}
+
+.header-first {
+  display: none;
+}
+
+.second {
+  background-image: url(./images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: contain;
+  vertical-align: top;
+  text-align: center;
+  justify-content: center;
+  flex-direction: column;
+  font-weight: 700px;
+  font-size: 30px;
+  color: #172b4d;
+  padding: 12px 12px 12px 15px;
+  font-family: 'Inter', sans-serif;
+  width: 345px;
+}
+
+.third {
+  font-style: normal;
+  font-weight: 400;
+  text-align: center;
+  line-height: 28px;
+  font-size: 20px;
+  vertical-align: top;
+  color: #172b4d;
+}
+
+.socials {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: flex-end;
+  list-style: none;
+  padding: 5;
+  gap: 20px;
+}
+
+.green {
+  color: #36b37e;
+}
+
+.line {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37e;
+}
```

Comment:
- [ ] Kindly add a border radius to this line, so it looks equal to the FIGMA design.

![image](https://user-images.githubusercontent.com/93630700/205329815-60dace8c-f86a-43a6-bc33-74b9ddb85717.png)